packages/ui-source-code-editor/package.json

```diff
@@ -26,24 +26,24 @@
     "@instructure/ui-babel-preset": "9.0.1",
     "@instructure/ui-test-queries": "9.0.1",
     "@instructure/ui-test-utils": "9.0.1",
-    "@testing-library/jest-dom": "^6.1.4",
-    "@testing-library/react": "^14.1.2"
+    "@testing-library/jest-dom": "^6.4.5",
+    "@testing-library/react": "^15.0.7"
   },
   "dependencies": {
-    "@babel/runtime": "^7.23.2",
-    "@codemirror/autocomplete": "^6.10.2",
-    "@codemirror/commands": "^6.3.0",
+    "@babel/runtime": "^7.24.5",
+    "@codemirror/autocomplete": "^6.16.0",
+    "@codemirror/commands": "^6.5.0",
     "@codemirror/lang-css": "^6.2.1",
-    "@codemirror/lang-html": "^6.4.6",
-    "@codemirror/lang-javascript": "^6.2.1",
+    "@codemirror/lang-html": "^6.4.9",
+    "@codemirror/lang-javascript": "^6.2.2",
     "@codemirror/lang-json": "^6.0.1",
-    "@codemirror/lang-markdown": "^6.2.2",
-    "@codemirror/language": "^6.9.1",
-    "@codemirror/legacy-modes": "^6.3.3",
-    "@codemirror/lint": "^6.4.2",
-    "@codemirror/search": "^6.5.4",
-    "@codemirror/state": "^6.3.0",
-    "@codemirror/view": "^6.21.3",
+    "@codemirror/lang-markdown": "^6.2.5",
+    "@codemirror/language": "^6.10.1",
+    "@codemirror/legacy-modes": "^6.4.0",
+    "@codemirror/lint": "^6.7.1",
+    "@codemirror/search": "^6.5.6",
+    "@codemirror/state": "^6.4.1",
+    "@codemirror/view": "^6.26.3",
```

Comment:
This went trough without bugs this time?

Reply:
yes and I didn't need to pin any packages this time. I'm not sure why it was failing previous times